op-chain-ops/cmd/op-upgrade-mcp/main.go

```diff
@@ -92,7 +92,7 @@ func main() {
 	oplog.SetGlobalLogHandler(log.NewTerminalHandler(os.Stderr, color))
 
 	app := &cli.App{
-		Name:  "op-upgrade",
+		Name:  "op-upgrade-mcp",
```

Comment:
Why rename the command? Does this break something?

Reply:
The other command is called `op-upgrade` - there are 2 of them, this one is going to be deleted, its one off code